lookup_tables/greynoise/basic/riot_basic.yml

```diff
@@ -14,6 +14,12 @@ LogTypeMap:
     - LogType: AlphaSOC.Alert
       Selectors:
         - "$.event.srcIP"
+    - LogType: Apache.AccessCombined
```

Comment:
@rleighton obviously not a complex fix but I wanted to get your 👍🏼  on the LUT and Pack yaml updates